rest/utilities_testing_resttester.go

```diff
@@ -66,6 +66,11 @@ func (rt *RestTester) GetDoc(docID string) (DocVersion, db.Body) {
 	return DocVersion{RevTreeID: *r.RevID}, body
 }
 
+func (rt *RestTester) TriggerOnDemandImport(docID string) {
+	// Trigger on-demand import for the document, don't care about response
```

Comment:
Adding a docstring allows this to be picked up by editors.

```suggestion
// TriggerOnDemandImport will use the REST API to trigger on an demand import via GET. This function intentionally does not check error codes in case the document does not exist or is invalid to be imported.
func (rt *RestTester) TriggerOnDemandImport(docID string) {
	// Trigger on-demand import for the document, don't care about response
```

Is there a use for not checking the error code? Because if this has an error code, it doesn't get imported.

There's mildly different code for GET `OnDemandImportForGet` and PUT `OnDemandImportForPut` which is why I noted this. I can't think of a reason that the GET version would fail. The `PUT` version can fail if there are conflicts, it would do an import of the document in the bucket, but then fail.